ibis/tests/strategies.py

```diff
@@ -178,34 +178,40 @@ def struct_dtypes(
     return dt.Struct(fields, nullable=draw(nullable))
 
 
-def geometry_dtypes(nullable=_nullable, srid=None):
+def geometry_dtypes(nullable=_nullable):
     return st.builds(
         dt.GeoSpatial,
         geotype=st.just("geometry"),
         nullable=nullable,
-        srid=st.just(srid),
+        srid=st.one_of(st.just(None), st.just(4321)),
```

Comment:
Is this to build the two separate strategies? If so, pretty neat : )

Reply:
Yep!